### PR TITLE
ignore __shelf__ label in stow task

### PR DESCRIPTION
**Pull request by knorth55**
waiting for upstream
required https://github.com/jsk-ros-pkg/jsk_recognition/pull/2151

- return `nil` when largest box is not found
- add `ignore_labels` `[0, 41]` in label_to_cpi
- use arc2017 object_segmentation_3d in stow task

### CODE REVIEW

File: jsk_apc2016_common/launch/object_segmentation_3d.launch [hunk 1]
```diff
@@ -41,8 +41,9 @@
           pkg="nodelet" type="nodelet"
           args="load jsk_pcl_utils/LabelToClusterPointIndices $(arg NODELET_MANAGER)">
       <remap from="~input" to="apply_context_to_label_proba/output/label" />
-      <rosparam>
+      <rosparam subst_value="true">
         bg_label: 0
```
Comment (wkentaro) [inline jsk_apc2016_common/launch/object_segmentation_3d.launch]: Please set to -1

File: jsk_apc2016_common/launch/object_segmentation_3d.launch [hunk 2]
```diff
@@ -41,8 +41,9 @@
...
+        ignore_labels: $(arg IGNORE_LABELS)
```
Comment (wkentaro) [inline jsk_apc2016_common/launch/object_segmentation_3d.launch]: You can just write `[0, 41]`.

File: jsk_arc2017_baxter/euslisp/lib/arc-interface.l [hunk 3]
```diff
@@ -222,15 +222,16 @@
...
+      (when box-msg
+        (dolist (box (send box-msg :boxes))
+          (when box
+            (let (volume)
+              (setq volume (* (send box :dimensions :x)
+                              (send box :dimensions :y)
+                              (send box :dimensions :z)))
+              (if (> bbox-volume-threshold- volume)
+                (return-from :get-largest-object-index
+                             (+ (send box :label) 1)))))))
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/arc-interface.l]: You can remove `+ ... 1`

File: jsk_apc2016_common/launch/object_segmentation_3d.launch (hunk 1 above)
Comment (knorth55) [inline jsk_apc2016_common/launch/object_segmentation_3d.launch]: updated

**Comment by knorth55** [comment]
@ 708yamaguchi this PR solves your problem.

**Pull request merged**